test/format.js

```diff
@@ -47,6 +50,8 @@ void (
     'qux\rquux',
     42,
     -42,
+    int(43),
+    int(-43),
```

Comment:
Let's take this out of the array, and use the `formatsPrimitive` macro only if `BigInt` is available. We should make sure the title clarifies we're testing a BigInt primitive.